docs/superpowers/plans/2026-03-28-chatbot-calendar-plan.md

```
@@ -0,0 +1,2002 @@
+    maxSteps: 5,
+  });
+
+  return result.toDataStreamResponse();
+}
+```
+
+- [ ] **Step 2: Add ANTHROPIC_API_KEY to env**
+
+Add to `.env`:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)